server/src/main/java/org/opensearch/common/cache/store/config/CacheConfig.java

```diff
@@ -113,6 +124,11 @@ public Builder<K, V> setWeigher(ToLongBiFunction<K, V> weigher) {
             return this;
         }
 
+        public Builder<K, V> setPolicyInfoWrapperFunction(Function<V, CachePolicyInfoWrapper> function) {
```

Comment:
Where is this being set ?

Reply:
This would be set within the CacheConfig that's sent to the TieredSpilloverCache factory or builder. Then the TieredSpilloverCache picks it up when it's constructed. There's an example (slightly different from the actual use case, since it's using String keys) on line 129 of TieredSpilloverCacheTests.java.

Reply:
Will that be part of separate PR to actually set it in the CacheConfig ?

Reply:
I thought it was going to be a future PR but it seems like Sagar's TieredSpilloverCache integration PR has gone in. I'll rebase from main and make those changes where it's actually constructed for use in the IndicesRequestCache.